packages/fxa-settings/src/components/Settings/HeaderLockup/en.ftl

```diff
@@ -4,5 +4,7 @@ header-menu-open = Close menu
 header-menu-closed = Site navigation menu
 header-back-to-top-link =
   .title = Back to top
+header-back-to-settings-link =
+  .title = Back to { -brand-mozilla } account settings
```

Comment:
```suggestion
  .title = Back to { -product-mozilla-account } settings
```

Reply:
@bcolsson I'd considered this, but figured it might lead to localization challenges with "settings" tacked on. Not a concern?

Reply:
@vpomerleau -this shouldn't be a problem. Locales that need to conjugate nouns have the ability to set the different cases in the brand term and select the appropriate version for the string. We already have one string `drop-down-menu-title-2` which is doing similar as well.